frontend/src/Explorer/View/Transaction.purs

```diff
@@ -70,7 +70,9 @@ summaryItems :: CTxSummary -> Language -> SummaryItems
 summaryItems (ctxSum@CTxSummary txSummary) lang =
     [ { label: translate (I18nL.tx <<< I18nL.txTime) lang
       , value: let  dateFormat = translate (I18nL.common <<< I18nL.cDateFormat) lang
-                    dateValue = fromMaybe noData <<< prettyDate dateFormat $ txSummary ^. ctsTxTimeIssued
+                    mDate      = txSummary ^. ctsTxTimeIssued
```

Comment:
Are these local vars `mDate`, `mDateValue really needed here (see my other comment https://github.com/input-output-hk/cardano-sl-explorer/pull/188/files#r127901432)? Compositing things has some power, but if you think `dateValue` was not readable or understandable before, we can do this change. Anyway, I don't prefer this new change, just two more lines of code...